src/hieroglyph/builder.py

```diff
@@ -35,6 +34,12 @@ def create(self, themename):
 else:
     from sphinx.theming import HTMLThemeFactory
 
+if sphinx.version_info >= (1, 5):
```

Comment:
I don't think there's any reason to support anything earlier than Sphinx 1.5 (which would be almost 3 years old at this point). Why don't we just update the import to the new location?